Extensions/blacklist.css

```diff
@@ -95,7 +95,7 @@
 
 .xblacklist_excuse {
```

Comment:
can you format this while you're in here? we should never have css properties on the same line